src/main/webapp/resources/angular/control/controlDonateCtrl.js

```diff
@@ -43,15 +43,24 @@
 
         };
 
-        self.saveDonate = function (donate, id) {
+        self.saveDonate = function (changeDonate, donate) {
             var donation = {
-                "id": id,
-                "donate": donate
+                "id": donate.id,
+                "donate": changeDonate
             };
-            ControlDonatesService.saveOwnDonate(donation).then(function (result) {
-                self.donates = result.data;
-                Notification({message: 'Donation updated successful', title: 'Notification'}, 'success');
-            })
+            if (ControlDonatesService.isDonateHaveProjectWithLess90PercentDonate(donate.percentDonateProject) ||
+                ControlDonatesService.isIncreaseDonation(changeDonate, donate.donateAmount)) {
+                ControlDonatesService.saveOwnDonate(donation).then(function (result) {
```

Comment:
- [ ] rename service to `donateService` in case it does not suppose to create new instances of donation object. General rule is to use variable naming convention in case of singleton object and class naming convention in case of classes that supposed to represent some entity and used with `new` keyword.